packages/runtime/test/integration/c2c-call/c2c-call.ts

```diff
@@ -91,61 +91,59 @@ describe("C2C call", function () {
 		);
 	});
 
-	// TODO: We don't handle the fee of inner group created by other tx in same group as well
-	// Should fix it in next patch release.
-	// https://www.pivotaltracker.com/n/projects/2452320/stories/181642443
-	// describe("Inner transaction in group", function() {
-	// 	it("should succeed", () => {
-	// 		const execParams: types.ExecParams = {
-	// 			type: types.TransactionType.CallApp,
-	// 			sign: types.SignType.SecretKey,
-	// 			fromAccount: alice.account,
-	// 			appID: firstApp.appID,
-	// 			foreignApps: [secondApp.appID],
-	// 			appArgs: ['str:call_method', `int:${1}`],
-	// 			payFlags: {
-	// 			totalFee: 1000
-	// 			}
-	// 		}
+	describe("Inner transaction in group", function () {
+		it("should succeed", () => {
```

Comment:
Maybe we should add a slightly longer explanation for the test case

Reply:
yes, I will add more detail about test cases.